### DISCUSSION TITLE
Line break in a Snackbar

### DISCUSSION THREAD

**Discussion by echdeneth** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3089] [Q&A] [answered]
In an MVVM scenario I assign the message of an exeption to the Snackbar: 
`MessageQueue.Enqueue(ex.Message);`
Unfortunately, only one line is displayed in the snackbar and the rest of the message is truncated, although a line break is possible. 
This also only displays one line:
`MessageQueue.Enqueue("First Line" + Environment.NewLine + "Second Line");`
What am I doing wrong?

**Comment by Keboo** [comment, marked as answer]
There is an attached property that you can set on the Snackbar instance itself to control the maximum height on the messages.
If you want to allow for any height, you can set it to `double.PositiveInfinity`. Something like this:

```xaml
<materialDesign:Snackbar materialDesign:SnackbarMessage.ContentMaxHeight="{x:Static sys:Double.PositiveInfinity}">
...
</materialDesign:Snackbar>
```
